onnxruntime/core/framework/session_options.h

```diff
@@ -111,7 +111,11 @@ struct SessionOptions {
 
   // By default the session uses its own set of threadpools, unless this is set to false.
   // Use this in conjunction with the CreateEnvWithGlobalThreadPools API.
+#if defined(__wasm__)
```

Comment:
@fs-eire do you suggest adding `&& __EMSCRIPTEN_PTHREADS__` here?

Reply:
yes